distribution/lib/Standard/Base/0.0.0-dev/package.yaml

```diff
@@ -18,22 +18,13 @@ component-groups:
           - Standard.Base.Data.list_directory
     - Web:
         exports:
-          - Standard.Base.Network.Http.new
-          - Standard.Base.Network.Http.fetch
-          - Standard.Base.Network.Http.get
-          - Standard.Base.Network.Http.post
-          - Standard.Base.Network.Http.post_form
-          - Standard.Base.Network.Http.post_json
-          - Standard.Base.Network.Http.put
-          - Standard.Base.Network.Http.put_json
-          - Standard.Base.Network.Http.head
-          - Standard.Base.Network.Http.options
+          - Standard.Base.Network.HTTP.HTTP.new
```

Comment:
These are the _component groups_ visible in component browsers! Yeah, simplifying the `Web` part is probably good idea.

Reply:
A start at tidying Web API. Will design and expand over next month or two.